js/scripts.js

```diff
@@ -0,0 +1,15 @@
+// plik scripts.js
+
+
+
+function rysujChoinke (j) {
```

Comment:
we use only english in code, so every function should be for example drawTree()
remove space before ()
name of parameters should show us what they are